source/atlas-cli-changelog.txt

```diff
@@ -12,6 +12,32 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.0.3:
+
+{+atlas-cli+} 1.0.3
+-------------------
+
+Released 28 April 2022
+
+Changes
+~~~~~~~
+
+- Improves :ref:`atlas-quickstart` commands:
+
+  - Clusters created using :ref:`atlas-quickstart` now start
+    with the cluster prefix and five digits.
+  - :ref:`atlas-quickstart` now displays a default cluster
```

Comment:
[optional nit] Is this worth capitalizing as the start of the bullet point? "Atlas" isn't capitalized in the other links so I don't know.
```suggestion
  - :ref:`Atlas quickstart <atlas-quickstart>` now displays a default cluster
```